ci.yml

```diff
@@ -1,7 +1,10 @@
 script:
-        - oc login -u kubeadmin -p mhk2X-Y8ozE-9icYb-uLCdV --insecure-skip-tls-verify=true https://api.crc.testing:6443
-        - IMAGE_BUILD_ARGS="--storage-driver vfs" ./build/build_ci.sh
-        - cd ./tests/smoketest 
-        - ./smoketest.sh
+        - export HOME=$(dirname "$PWD")
+        - oc login -u kubeadmin -p BdTY8-Vra64-gD3cI-dAPS6 --insecure-skip-tls-verify=true https://api.crc.testing:6443
+        - oc new-project "${OCP_PROJECT}"
+        - ./build/build.sh
+        - ./build/deploy_ci.sh
```

Comment:
I think you want `./build/build_ci.sh` here.

Reply:
Nevermind. I did look initially for a filename change, but missed it. I see it now.